axiom/nr_php_packages.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <math.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "nr_axiom.h"
+#include "nr_limits.h"
+#include "nr_php_packages.h"
+#include "util_memory.h"
+#include "util_minmax_heap.h"
+#include "util_vector.h"
+#include "util_hashmap.h"
+#include "util_hashmap_private.h"
+#include "util_strings.h"
+
+nr_php_package_t* nr_php_package_create(char* name, char* version) {
+  nr_php_package_t* p = NULL;
+
+  if (name == NULL || version == NULL) {
+    return NULL;
+  }
+
+  p = (nr_php_package_t*)nr_malloc(sizeof(nr_php_package_t));
```

Comment:
Check if `p` is not `NULL`.

Reply:
Added: [89dce80](https://github.com/newrelic/newrelic-php-agent/pull/754/commits/89dce80cf7bed926fcf047446043fa37c1713e30)

Reply:
[New Relic PHP Agent's style guide](https://github.com/newrelic/newrelic-php-agent/blob/main/STYLEGUIDE.md) requires use of [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions).